src/components/flowchart/flowchart.js

```diff
@@ -220,22 +220,28 @@ export class FlowChart extends Component {
 
     if (changed('edges', 'nodes', 'layers', 'chartSize', 'clickedNode')) {
       // Don't zoom out when the metadata or code panels are opened or closed
-      if (prevProps.visibleMetaSidebar !== this.props.visibleMetaSidebar) {
+      const metaSidebarChanged =
```

Comment:
[Nit] if this is to track visibility of sidebar then `const metaSidebarViewChanged` or `const metaSidebarVisibilityChanged` might fit